src/SFA.DAS.Campaign.Web/Views/Apprentice/Index.cshtml

```diff
@@ -1,8 +1,59 @@
-﻿@{
-    ViewData["Title"] = "Apprentice";
+﻿@using Microsoft.AspNetCore.Html
+@using SFA.DAS.Campaign.Web.Constants
+@using SFA.DAS.Campaign.Web.ViewComponents.Modal
+@{
+    ViewData["Title"] = "Ignite your career";
 }
 
-<div style="height: 100px;"></div>
 
-<h1 class="heading-xl">Apprentice</h1>
+<div class="page">
+
+    <h2 class="heading-l">Introduction to apprenticeships</h2>
+
+    <p>An apprenticeship gives you hands-on experience, a salary and the opportunity to train while you work.   You get a same as other employees: a contract of employment, holiday leave and sick pay.   Anyone from the age of 16 can become an apprentice as long as they're living in England and not in full-time education.</p>
+    <p>An apprenticeship is a real job. It's a way to:</p>
+
+    <ul class="list list--bullet list--bullet-apprentice">
+        <li>get paid and learn at the same time, with at least 20% of your time set aside for learning - normally at a college, university or other training providers</li>
+        <li>gain official certification of your skills, equivalent to a qualification ranging from level 2 (GCSE equivalent) up to degree level</li>
+        <li>start a long-term career path with the possibility of higher earnings</li>
+    </ul>
+
+
+    <p>An apprenticeship can take 2 to 4 years to complete depending on which level you do.   </p>
+
+    <p>By successfully completing your apprenticeship, you'll have gained the confidence and skills needed to progress in your chosen career. </p>
+
+    <p>Apprenticeships are available across a wide range of industries and organisations, from local businesses to large national brands.</p>
+
+    <p>Apprenticeships have been through a massive change in the last few years.  They're not the 'easy option' and studying whilst you're working takes hard work!  Degree apprenticeship are now recognised as credible alternatives to full-time university courses. </p>
+
+    <p>If you live in <a href="#"> Scotland</a>, <a href="#">Wales</a>  or <a href="">Northern Ireland</a>  you can find out more about apprenticeships in your area.  </p>
+
+
+    <div class="next-step-link">
+        <a href="">
+            Benefits
+            <span class="next-step-link__subtitle">Next step</span>
+        </a>
+    </div>
+
+</div>
+
+@{
+
+    IHtmlContent HeroHeadingContent() => Helper.Body(@<ul class="list list--arrows">
+                                                       <li><a href="#" class="hero__link">How much can you earn</a> </li>
+                                                       <li><a href="#" class="hero__link">Cost to you</a> </li>
+                                                       <li><a href="#" class="hero__link">Future prospects</a> </li>
+                                                       <li><a href="#" class="hero__link">Top 5 reasons for becoming an apprentice</a> </li>
```

Comment:
i would suggest omitting all links until the destination has been defined, to avoid anything being missed.